demo/index.html

```diff
@@ -172,12 +172,19 @@
         </div>
         <div>
           <label for="drmSettingsVideoRobustness">Video Robustness:</label>
-          <input id="drmSettingsVideoRobustness" type="text" class="flex-grow">
+          <input id="drmSettingsVideoRobustness" type="text" class="flex-grow" list="robustnessSuggestions">
         </div>
         <div>
           <label for="drmSettingsAudioRobustness">Audio Robustness:</label>
-          <input id="drmSettingsAudioRobustness" type="text" class="flex-grow">
-        </div>
+          <input id="drmSettingsAudioRobustness" type="text" class="flex-grow" list="robustnessSuggestions">
+        </div>
+        <datalist id="robustnessSuggestions">
```

Comment:
Even though I would like to use this, [caniuse](http://caniuse.com/#search=datalist) says it is not well supported in all the browsers we support.  You'll probably need to do this in JavaScript.

Reply:
It will probably be fine.

The only browser with no support on that list is Safari, at which point Safari behaves the same as it did before.

Partial support on Firefox refers to text fields only, which works here.

Partial support on IE/Edge is a little more concerning, since it seems that some events may not fire.  But if we don't need those events, it may still work.  (I don't think we need change events on these fields.)